velox/common/caching/SsdFile.cpp

```diff
@@ -739,11 +755,13 @@ void SsdFile::checkpoint(bool force) {
       const auto mapMarker = kCheckpointMapMarker;
       state.write(asChar(&mapMarker), sizeof(mapMarker));
       for (auto& pair : entries_) {
-        auto id = pair.first.fileNum.id();
+        const auto id = pair.first.fileNum.id();
         state.write(asChar(&id), sizeof(id));
         state.write(asChar(&pair.first.offset), sizeof(pair.first.offset));
-        auto offsetAndSize = pair.second.bits();
+        const auto offsetAndSize = pair.second.fileBits();
         state.write(asChar(&offsetAndSize), sizeof(offsetAndSize));
+        const auto checksum = pair.second.checksum();
```

Comment:
Can we make checksum write an option if not let's run shadow for a while to make sure that checksum write doesn't cause significant regression? Thanks!